app/views/identified_clients/index.xlsx.axlsx

```diff
@@ -0,0 +1,75 @@
+wb = xlsx_package.workbook
+wb.add_worksheet(name: client_type) do |sheet|
+  title = sheet.styles.add_style(sz: 12, b: true, alignment: {horizontal: :center})
+  sheet.add_row(
+    [
```

Comment:
Use `download_headers` (or the keys if made into a map)?

Reply:
Ah, this file should have been removed.